docs/iot-devkit/devkit-get-started.md

```diff
@@ -32,27 +32,46 @@ To connect the IoT DevKit to your computer:
 
 ![Hardware connections](media/iot-devkit-get-started/connect.jpg)
 
-## Configure Wi-Fi
+## Get Started with Azure IoT Hub
 
-IoT projects rely on internet connectivity. Use AP Mode on the DevKit to configure and connect to Wi-Fi.
+1. Follow this [tutorial](https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-node#create-an-iot-hub) to
+	- Create a new Azure IoT Hub from portal.
+	- Register a new device into Azure IoT Hub and make a note of the device connection string, which looks like:
+	`HostName={YourIoTHubName}.azure-devices.net;DeviceId=MyNodeDevice;SharedAccessKey={YourSharedAccessKey}`
 
-1. Hold down button **B**, push and release the **Reset** button, and then release button **B**. Your IoT DevKit enters AP mode for configuring the Wi-Fi connection. The screen displays the service set identifier (SSID) of the DevKit and the configuration portal IP address:
+	>Please follow the section of **Create an IoT hub** and **Register a device** only. Please skip the section to run the simulated application.
+
+1. Download the latest version of [GetStarted firmware](https://aka.ms/devkit/prod/getstarted/latest) for IoT DevKit which includes the bootloader.
+
+1. Drag and drop the `.bin` file you downloaded from step 1 into the disk named **AZ3166** and wait for IoT Devkit to restart.
+
+	![copy binary](media/copy_file.png)
+
+1. IoT projects rely on internet connectivity. Use AP Mode on the DevKit to configure and connect to Wi-Fi. 
+	Hold down button **B**, push and release the **Reset** button, and then release button **B**. Your IoT DevKit enters AP mode for configuring the Wi-Fi connection. The screen displays the service set identifier (SSID) of the DevKit and the configuration portal IP address:
 
     ![Reset button, button B, and SSID](media/iot-devkit-get-started/wifi-ap.jpg)
 
-2. Use a Web browser on a different Wi-Fi enabled device (computer or mobile phone) to connect to the IoT DevKit SSID displayed in the previous step. If it asks for a password, leave it empty.
+1. Use a Web browser on a different Wi-Fi enabled device (computer or mobile phone) to connect to the IoT DevKit SSID displayed in the previous step. If it asks for a password, leave it empty.
 
     ![Network info and Connect button](media/iot-devkit-get-started/connect-ssid.png)
 
-3. Open **192.168.0.1** in the browser. Select the Wi-Fi network that you want the IoT DevKit to connect to, type the password for the Wi-Fi conection, and then click **Connect**.
+1. Open **192.168.0.1** in the browser. Select the Wi-Fi network that you want the IoT DevKit to connect to, type the password for the Wi-Fi conection and input the device connection string you retrieved in step 1. Then click **Connect**.
```

Comment:
consider to mention people copy the connection string first, before connect to the board